docs/readthedocs/source/_static/js/chronos_tutorial.js

```diff
@@ -166,12 +181,64 @@ $(".checkboxes").click(function(){
             disCheck(disIds);
         }
     }
+
+    displaySelected();
 });
 
-//event when click the tags' buttons
-$("details p button").click(function(){
-    var id = $(this).val();
-    $("#"+id).trigger("click");
+//event when choosing forecasters
+$(".forecasters").click(function(){ 
+    //reset all the details 'forecasterSelected' class
+    $('#tutorial details').each(function(){
+        $(this).removeClass('forecasterSelected');
+    });
+
+    //get the selected forecasters
+    var forecasters = [];
+    $('.forecasters:checked').each(function(){
+        forecasters.push($(this).val());
+    });
+
+    if(forecasters.length===0){
+        //no forecaster is checked, display all
+        $('#tutorial details').each(function(){ 
+            $(this).addClass('forecasterSelected');
+        });
+        $(".showingForecaster i").text("All Forecasters");
+    }
+    else{
+        //mark selected forecasters
+        $('#tutorial details').each(function(){
+            var sons = [];
+            $(this).find(".roundbutton").each(function(){
+                sons.push($(this).val())
+            });
+
+            if(forecasters.every(val => sons.includes(val))){
+                $(this).addClass('forecasterSelected');
+            }
+        });
+
+        let fs = "";
+        for(var i=0;i<forecasters.length;i++){
+            fs += forecasters[i]+',';
+        }
+        fs = fs.substring(0, fs.length - 1);
```

Comment:
@TheaperDeng Just a tiny concern. Seems like it would be better here to add one more space after `,` to avoid cases like this
![image](https://user-images.githubusercontent.com/54161268/199632136-4af06a5a-ac57-4c4a-9286-eacab8ddd335.png)
That is , changing line 223 to `fs += forecasters[i]+', ';` and line 225 to `fs = fs.substring(0, fs.length - 2);`

Others LGTM.